src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolverTest.php

```diff
@@ -285,7 +285,7 @@ public function testGetSessionMissMatchOnNull()
         self::getResolver()->getArguments($request, $controller);
     }
 
-    public function testPinnedResolver()
+    public function testTargetedResolver()
```

Comment:
There are multiple `Pinning` remaining in file

Reply:
Oops, fixe